tools/build_java_api_docs.py

```diff
@@ -48,6 +49,18 @@
 TOOLS_DIR = pathlib.Path(__file__).resolve().parent
 REPO_ROOT = TOOLS_DIR.parent
 
+
+def checkout_ndarray():
+  repo_url = 'https://github.com/tensorflow/java-ndarray'
+  local_repo_path = REPO_ROOT/'ndarray'
+  if not pathlib.Path(local_repo_path).exists():
+    local_repo = Repo.clone_from(repo_url, local_repo_path)
+  else:
+    local_repo = Repo(local_repo_path)
+  local_repo.remotes['origin'].fetch()
+  local_repo.git.checkout('v1.0.0')
```

Comment:
Maybe this should be defined up top as `TF_NDARRAY_VERSION` so it's easier to see?